From 89af5c84d4b920d22ae882fcc5ab71ba09443cea Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 30 Dec 2016 18:28:18 +0200
Subject: [PATCH 28/30] Use backports.lzma, not lzma

This is needed when using Python 2.x and should be dropped
after moving to Python 3.x

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 dnf/yum/misc.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dnf/yum/misc.py b/dnf/yum/misc.py
index 441b0148..6b4aea0a 100644
--- a/dnf/yum/misc.py
+++ b/dnf/yum/misc.py
@@ -37,7 +37,7 @@ import gpgme.editutil
 import gzip
 import hashlib
 import io
-import lzma
+import backports.lzma
 import os
 import os.path
 import pwd
-- 
2.11.0

